python/llm/dev/benchmark/pipelines/llama2_test.py

```diff
@@ -0,0 +1,94 @@
+#
```

Comment:
We may need to better design this. It is not convenient to add one test py for each model.
Probably one naive way is to add benchmarkwrapper using commands (e.g. sed), grep the output time and do the comparison in bash file?

Reply:
It would be great to use one script to test all the models, but they don't have the same inputs, and some like whisper even needs extra dependencies and datasets...

Reply:
I mean probably we can use one script like testing examples, and in the script we run the generate.py of all models, grep the result and do the perf test?

Reply:
Yeah this is possible